### PR TITLE
add PortaGloryhole.yml

### PR DESCRIPTION
**Pull request by JaseNZC**

### CODE REVIEW

File: scrapers/PortaGloryhole.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    url:
+      - portagloryhole.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+     Title:
+        selector: //h2
+     Performers:
+       Name: //h5[1]/a
```
Comment (bnkai) [inline scrapers/PortaGloryhole.yml:13]: using
```yaml
       Name: //h5[i[@class="icon-female"]]/a
```
seems safer

File: scrapers/PortaGloryhole.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+     Tags:
+       Name: //h5[@class="video_categories"]/a
+     Details: //div[@id="viddesc145"]/p
```
Comment (bnkai) [inline scrapers/PortaGloryhole.yml:16]:
```yaml
     Details: //div[@class="desc"]/p
```
probably safer

**Comment by JaseNZC** [comment]
Done.

**Comment by JaseNZC** [comment]
Now it should be ok.

**Pull request merged**